ApplicationView/ApplicationController.cs

```diff
@@ -981,9 +981,34 @@ public void OnLoadActions()
 					PrinterConnection.Instance.ConnectToActivePrinter();
 				}, 2);
 			}
+
```

Comment:
This code should move back to PrinterCommunication and have the oneOrMoreValuesReset and ReloadAdvancedControlsPanel logic removed